apps/cbportal/base/cloudbroker__cloudspace/methodclass/cloudbroker_cloudspace.py

```diff
@@ -108,6 +108,10 @@ def moveVirtualFirewallToFirewallNode(self, cloudspaceId, targetNid, **kwargs):
         if cloudspace.status != 'DEPLOYED':
             raise exceptions.BadRequest('Could not move fw for cloudspace which is not deployed')
 
+        node = self.syscl.node.get(targetNid)
+        if not node.active:
+            raise RuntimeError("Node is not active")
```

Comment:
this should be exceptions.BadRequest not runtimeError so it gets translated in a proper 400